pkg/resource/promtail-wiring/reconciler.go

```diff
@@ -28,6 +28,8 @@ type Reconciler struct {
 // ReconcileCreate ensure user value configmap is set in observability bundle
 // for the given cluster.
 func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Interface) (ctrl.Result, error) {
+	isWC := lc.GetClusterName() != lc.GetInstallationName()
```

Comment:
Can you check @marieroque PR and do the same?

Reply:
Sure :)

Reply:
Done